### DISCUSSION TITLE
glitch / screech in MP3 recordings

### DISCUSSION THREAD

**Discussion by charlie-foxtrot** [rtl-airband/RTLSDR-Airband#205] [General]
Every so often I get a "glitch" in the MP3 recording.  Some times its across both channels on a mixer output, sometimes its only one.  In all cases it sounds like a loud screech when you listen to the audio.  I was hoping #202 would fix it but did not.

I've looked at all the stats I'm logging (CPU, noise floor, overruns, etc) while this happens and nothing is suspicious.

Has anyone else seen this?  Any ideas?

See below for an example of two MP3 mixer outputs showing this.  In both cases I have some frequencies going to the left channel and other frequencies going to the right channel.

<img width="1093" alt="Screen Shot 2020-12-15 at 5 42 09 PM" src="https://user-images.githubusercontent.com/13514783/102294282-ceaca100-3efd-11eb-848b-7015304f3398.png">

**Comment by clam-i-am** [comment]
@charlie-foxtrot I've got no ideas whatsoever, but wanted to chime in that I've heard this in my mp3 recordings - many times.  Common enough that I've just gotten used to putting such digital rip/glitch noises mentally in the background.

**Reply by charlie-foxtrot** [reply]
Glad I'm not alone ;)

I'm seeing this using a raspberry pi 4 with 3 dongles, all doing NFM signals and running through mixers.  Is your setup different in any way that could help narrow it down?

**Reply by clam-i-am** [reply]
pi4, 2 dongles, each being fronted by an LNA. Used to be a pi3b+, same issue.

I _believe_ that I tend to hear the glitch at the beginning of transmissions with high signal strength - especially short ones where it's just a mic click or a sub-second reply... just an observation over time on this end, I wouldn't treat that as a truism without any other evidence.

**Reply by clam-i-am** [reply]
Doing AM only for ATC, mixers and standalone.  Attached is an example.  You'll hear 3 rips - first just before 3s, second around 3.5s, and the third just before 6.5s.

**Reply by clam-i-am** [reply]
@charlie-foxtrot just an update, I've listened with more intent the past few days on the live stream as opposed to the mp3 recordings (what I usually listen to). I only hear the digital glitches during the mp3s.  Live stream is fine.

I do have `continuous = false;` in my config to save on disk space and excessive silence.  Not sure if that is related, but a thought.
